index.bs

```diff
@@ -322,6 +322,12 @@ every point in the <a>node impact region</a> of any {{Node}} in the
 The <dfn export>impact fraction</dfn> of a {{Document}} |D| is the area of the
 <a>impact region</a> divided by the area of the <a>viewport</a>.
 
+NOTE: Computing the area of the impact region is an instance of the
+<a href="https://en.wikipedia.org/wiki/Klee's_measure_problem">Klee measure
```

Comment:
This link does not work, it should be https://en.wikipedia.org/wiki/Klee%27s_measure_problem